<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Selection</class>
    <widget class="QWidget" name="Selection">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>753</width>
                <height>542</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                        <string>Selected Values</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QTableWidget" name="selection_table">
                                <property name="selectionBehavior">
                                    <enum>QAbstractItemView::SelectRows</enum>
                                </property>
                                <column>
                                    <property name="text">
                                        <string>X</string>
                                    </property>
                                </column>
                                <column>
                                    <property name="text">
                                        <string>Y</string>
                                    </property>
                                </column>
                                <column>
                                    <property name="text">
                                        <string>Value</string>
                                    </property>
                                </column>
                            </widget>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QPushButton" name="import_button">
                                        <property name="text">
                                            <string>Import</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="export_button">
                                        <property name="text">
                                            <string>Export</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="clear_button">
                                        <property name="text">
                                            <string>Clear</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox_2">
                    <property name="title">
                        <string>Style</string>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                        <item row="0" column="1">
                            <widget class="QColorButton" name="marker_color">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QColorButton" name="text_color">
                                <property name="text">
                                    <string/>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="0">
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>Marker Colour:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>Text Colour:</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QColorButton</class>
            <extends>QPushButton</extends>
            <header>solarviewer.ui.util</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
